zerver/lib/error_notify.py

```diff
@@ -1,3 +1,4 @@
+# System documented in https://zulip.readthedocs.io/en/latest/logging.html
```

Comment:
Ah, good thought.

Should these notices also go on `zerver/lib/report.py` and `static/js/blueslip.js`?

Reply:
already in report.py, but added to blueslip.